HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -271,8 +271,17 @@ def self.apply_ground_to_air_heat_pump(model, runner, weather, heat_pump,
     # Supplemental Heating Coil
     htg_supp_coil = create_supp_heating_coil(model, obj_name, heat_pump)
 
+    # Site Ground Temperature Undisturbed
+    ts_amp_1, ts_amp_2, pl_1, pl_2 = Location.get_xing_amplitudes(weather.header.Latitude, weather.header.Longitude)
+    xing = OpenStudio::Model::SiteGroundTemperatureUndisturbedXing.new(model)
+    xing.setSoilDensity(920)
```

Comment:
Should we use the Xing soil density default value here (962) instead of setting the Kusuda default value?